_pages/dev/icon-library.md

```diff
@@ -76,15 +72,22 @@ ConfigModule.withConfig(<IconConfig>{
 });
 ```
 
-### SVG based icons
-In order to construct the SVG icon, the component will create an `xlink` to the SVG symbol. If a `url` is given for the icon type, the symbol will be constructed using the url and symbol configuration. The above setup, will produce links to `./assets/sprite.svg#info` and `./assets/sprite.svg#warning`.
+The configuration above uses a setup that combines Bootstrap 3 Glyphicons, Font Awesome icons (both fas and fab), and SVG icons.
+
+The Font Awesome icon set provides ready-to-use icons with defined classes, such as `fab fa-cc-visa` for the VISA icon. You can search for icons and find their related class definitions on the [Font Awesome website](https://fontawesome.com/icons).
+
+### SVG-Based Icons
+
+To construct an SVG icon, the component creates an `xlink` to the SVG symbol. If a `url` is given for the icon type, the symbol is constructed using the URL and the symbol configuration. The setup shown in the previous section produces links to `./assets/sprite.svg#info` and `./assets/sprite.svg#warning`.
+
+**Note:** To generate a font sprite for Font Awesome, you can use the [fontawesome-svg-sprite-generator](https://github.com/Minecrell/fontawesome-svg-sprite-generator). This library can automatically generate a sprite with the Font Awesome icons that you are using in your storefront. It is also possible to give the icons a custom icon name.
+
+### Font-Based Icons
 
-**Note**: to generate a font sprite for font awesome, you can use https://github.com/Minecrell/fontawesome-svg-sprite-generator. This is a convenient library to automically generate a sprite with the icons that you use in the storefront. Optionally, the icons can have a custom icon name. 
+To support font-based icons, the component simply adds the icon symbol to the list of style classes. The following is an example:
 
-### Font based icons
-To support font based icons the component simply adds the icon symbol to the list of style classes, for example 
 ```html
 <cx-icon class="fas fa-shopping-cart"></cx-icon>
 ```
 
-In order to load the font related CSS file, a resource can be added for a specific icon type or for all (by leaving out types entirely). When a resource is added, the CSS file be *linked* to the DOM dynamically. Spartacus will make sure that the font is only loaded once. 
+To load the font-related CSS file, a resource can be added for a specific icon type, or for all types (by leaving out types entirely). When a resource is added, the CSS file is linked to the DOM dynamically. Spartacus ensures that the font is only loaded once.
```

Comment:
Can we add the following behind:

> As stated before in this document, Spartacus is configured with an icon configuration which is using font awesome icons. This is added in the `IconModule` directly with the `fontawesomeIconConfig`.

Alternatively, we could rewrite and move the section regarding spartacus using fonts here.
> Spartacus is shipped as a library... ....more icons than are needed for a storefront.

Reply:
I decided to rewrite the earlier section, as follows:

Spartacus is shipped as a library, so there is no straightforward approach to loading SVG files. Libraries are installed in the `node_modules` folder, and using SVG-based icons would require an import from an application location, such as the `/assets` folder. For this reason, Spartacus is configured by default to use a font-based setup for icons. This default configuration uses Font Awesome icons, which are added to the `IconModule` directly with the `fontawesomeIconConfig`. Note, this is not necessarily the best practice for production, because most standard font icon sets cover more icons than are needed for a storefront.

Let me know if this is good for you 🙂